documentation/developer-guide/modules/tooling-guide/pages/bamm-cli.adoc

```diff
@@ -8,6 +8,56 @@ The BAMM CLI is a command line tool for the validation of Aspect models and the
 
 TIP: Download latest version: icon:download[] https://github.com/OpenManufacturingPlatform/sds-sdk/releases/download/v{sds-sdk-version}/bamm-cli-{sds-sdk-version}.jar[bamm-cli-{sds-sdk-version}.jar]
 
+== Preconditions
+=== Step 1 - Java Download
+The BAMM CLI SDK may be downloaded as an executable .jar or it may be built from the user individually. In case of an individual built a java development kit (JDK) is required. To run the java application a JRE is enough. 
+You can download the JDK from https://adoptium.net/?variant=openjdk11&jvmVariant=hotspot[here].
+
+image::../images/1_JDK_Download.png[]
+
+=== Step 2 - Java Check
+Ensure your Java version is up to date:
+1.	Open a terminal or command line
+2.	Enter “java -version”
+3.	Check if result is identical to the downloaded version
+
+image::../images/2_JDK_VersionControl.png[]
+
+=== Step 3 - BAMM Download
+Download the released https://github.com/OpenManufacturingPlatform/sds-sdk/releases[BAMM CLI SDK]
+
+1. In case you want to run a built executable -> Download only the first file "bamm-cli-1.02.jar" (Note: Version number may change over time)
+2. In case you want to build your own executable -> Clone the repository and build with https://maven.apache.org[Maven].
+
+image::../images/3_BAMM_CLI_Download.png[]
+
+=== Step 4 - BAMM Placement
+1. Create directory (e.g. on Desktop)
+2. Place downloaded .jar file in the folder.
+
+image::../images/4_CreateFolderStructure.png[]
+
+3. Also place example model in the corresponding folder. 
+4. !Remember: the model file must be placed according to the URN-schema of your model.
+In this example it's: @prefix : <urn:bamm:io.openmanufacturing:1.0.0#>. That implies the current position of the model is false.
```

Comment:
Surround the code part (starting with "@" and ending with "#>") with backticks (\`...\`), this will turn it from "regular text" to "`code highlighted text`"